samples/core-projects/console-apps/NewTypes/test/NewTypesTests/project.json

```diff
@@ -1,22 +1,28 @@
 {
   "version": "1.0.0-*",
-  "testRunner": "xunit",
-
+  "buildOptions": {
+    "debugType": "portable"
+  },
   "dependencies": {
-    "Microsoft.NETCore.App": {
-      "type":"platform",
-      "version": "1.0.0"
-    },
-    "xunit":"2.2.0-beta2-build3300",
-    "dotnet-test-xunit": "2.2.0-preview2-build1029",
-    "NewTypes": "1.0.0"
+    "System.Runtime.Serialization.Primitives": "4.1.1",
```

Comment:
It's generated by the template: https://github.com/dotnet/cli/blob/rel/1.0.0-preview2/src/dotnet/commands/dotnet-new/CSharp_xunittest/project.json.template